### PR TITLE
Community 2.0

### PR DESCRIPTION
**Pull request by sumitsaurabh927**
Off we go! 🌊🚀

### CODE REVIEW

File: community/get-involved.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+### Help in onboarding and answering questions.
+
+- Welcome new users into the community
+- Answer questions in the [community support channel]([link](https://discord.com/channels/895029566685462578/1019663407915483176)).
+- Offer input and opinions about various solutions you have tried.
+
+### Improve our documentation
+
+- Check out our[Github repo](https://github.com/novuhq/docs)
```
Comment (unicodeveloper) [inline community/get-involved.mdx:67]: Github => GitHub

File: community/get-involved.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    - Updating outdated examples
+    - Correcting typos and language for clarity
+    - Find and fix broken links, etc.
+
+
+### Help with the SDKs
+
+- Link to contributors guide(https://github.com/novuhq/novu/blob/next/CONTRIBUTING.md)
+- We have SDKs in various languages and frameworks, most written and maintained by community members.
+- You can either become a maintainer there or help the existing maintainers by bringing the SDK up to speed with the latest features present in the core product. (name and link all SDKs).
```
Comment (unicodeveloper) [inline community/get-involved.mdx:79]: (name and link all SDKs) - I believe you are meant to add them here.

File: community/get-involved.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Create content
+
+- Content writers program: Apply to become a content writer with us and get paid to write about use-cases highlighting how you solved a problem using Novu
```
Comment (unicodeveloper) [inline community/get-involved.mdx:83]: Is there a link where they can apply? None is added here. @sumitsaurabh927

**Review by unicodeveloper — COMMENTED** [review]
Nice work! Please check the comments.

File: community/get-involved.mdx (hunk 2 above)
Comment (sumitsaurabh927) [reply community/get-involved.mdx:79]: Yeah, my bad! 🙈🙈🙈

File: community/get-involved.mdx (hunk 3 above)
Comment (sumitsaurabh927) [reply community/get-involved.mdx:83]: Not at the moment. I'll update it once we do

**Review by unicodeveloper — APPROVED** [review]

**Pull request merged**